src/main/webapp/app/pages/newsPage/NewsPageContent.tsx

```diff
@@ -132,6 +135,16 @@ export const CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS = [
   { name: EVIDENCE },
 ];
 
+export const CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS_NEW = [
```

Comment:
What's the difference of using CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS?

Reply:
The only difference is the column name that we use:
New `Drug(s) currently in OncoKB   |  Drug(s) added to OncoKB`
vs.
Old `Drug(s) Already in OncoKB   |   Newly Added Drug(s)`

Reply:
The team wants to use the new language for the header names and thinks its fine to keep the old language for the older releases.